src/Views/FirmwareView.vala

```diff
@@ -376,8 +376,8 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
 
     private void show_shutdown_dialog () {
         var message_dialog = new Granite.MessageDialog.with_image_from_icon_name (
-            _("An update requires the system to shut down to complete"),
-            _("This will close all open applications and turn off this device."),
+            _("Installing new firmware requires the system to be turned off to complete"),
```

Comment:
I think we should probably stick with "shut down" since that's the name of the menu item in the power menu so it's more explicitly linked to something you may be familiar with. I agree that "turned off" sounds more natural, but in this specific case it might be slightly less clear